### ISSUE TITLE
BUG: Short SLTP bracket prices inverted in offline env — SL/TP swapped for all short positions

### ISSUE THREAD

**Issue by BY571** [TorchTrade/torchtrade#149]
## Summary

Short positions in the offline SLTP environments (`SequentialTradingEnvSLTP`, `SeqFuturesSLTPEnv`) have their stop-loss and take-profit bracket prices **completely inverted**. The stop-loss triggers on favorable price moves and the take-profit triggers on unfavorable ones. This means every policy trained offline with short SLTP actions learns inverted risk/reward, and will behave catastrophically when deployed to live trading where bracket prices are correct.

## Root Cause

There are **three separate implementations** of SLTP action map building that disagree on short bracket semantics:

1. **Live envs** (`torchtrade/envs/utils/action_maps.py:82`) — `create_sltp_action_map` **swaps** SL/TP for shorts:
   ```python
   ("short", tp, sl)  # Swapped: stores (side, tp_pct, sl_pct)
   ```

2. **Offline envs** (`torchtrade/envs/offline/sequential_sltp.py:206`) — `_build_action_map` does **NOT swap**:
   ```python
   ("short", sl, tp)  # NOT swapped: stores (side, sl_pct, tp_pct)
   ```

3. The offline env then passes these unswapped values to `calculate_short_bracket_prices` (`torchtrade/envs/utils/sltp_helpers.py:31-60`), which expects already-swapped values per its docstring.

## Concrete Example

Entry price: **50,000**, SL level: 2.5%, TP level: 5%

### Live env (CORRECT)
Action map stores `("short", 0.05, -0.025)` (swapped). Then:
- `stop_loss = 50000 * (1 + 0.05) = 52,500` — **above entry** ✅
- `take_profit = 50000 * (1 + (-0.025)) = 48,750` — **below entry** ✅

### Offline env (WRONG)
Action map stores `("short", -0.025, 0.05)` (not swapped). Passed to `calculate_short_bracket_prices`:
- `stop_loss = 50000 * (1 + (-0.025)) = 48,750` — **below entry** ❌ (should be above)
- `take_profit = 50000 * (1 + 0.05) = 52,500` — **above entry** ❌ (should be below)

The SL and TP are completely swapped. The "stop-loss" triggers when price drops (favorable for shorts) and the "take-profit" triggers when price rises (unfavorable for shorts).

## Impact

- **All offline SLTP short training is wrong** — agents learn inverted bracket behavior for shorts
- **Offline-to-live policy transfer is broken** — same action index produces different bracket prices
- Affects `SequentialTradingEnvSLTP` and any env using the offline `_build_action_map`

## Affected Files

- `torchtrade/envs/offline/sequential_sltp.py` — `_build_action_map` (line ~206)
- `torchtrade/envs/utils/action_maps.py` — `create_sltp_action_map` (line ~82)
- `torchtrade/envs/utils/sltp_helpers.py` — `calculate_short_bracket_prices` (lines 31-60)
- `torchtrade/envs/offline/infrastructure/utils.py` — third copy of action map logic (lines 84-167)

## Suggested Fix

**Option A**: Make the offline `_build_action_map` swap SL/TP for shorts, matching the live env:
```python
# In sequential_sltp.py _build_action_map, for short actions:
("short", tp, sl)  # Swap to match live env convention
```

**Option B**: Consolidate all three action map implementations into a single shared function (eliminates the root cause — code duplication).

Option B is preferred as it prevents this class of bug from recurring.

## Reproduction

Open a short position in `SequentialTradingEnvSLTP` with any SL/TP levels and inspect `self.stop_loss` and `self.take_profit`. The SL will be below entry price (should be above) and TP will be above (should be below).
